tests/testthat/test-bug-501-rectangles.R

```diff
@@ -0,0 +1,15 @@
+
+test_that("Edges stop at outside of rectangle node", {
+  rectangle_edges <- function() {
+    g <- graph(c(1,2, 1,4, 2,1, 2,5, 2,3, 4,1, 5,2, 3,2))
```

Comment:
```suggestion
    g <- make_graph(c(1,2, 1,4, 2,1, 2,5, 2,3, 4,1, 5,2, 3,2))
```

`graph` happens to be another deprecated/undocumented construct :-)